fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2CheckCompatibility.java

```diff
@@ -18,15 +18,14 @@
 import com.ibm.fhir.database.utils.version.SchemaConstants;
 
 /**
- * @author rarnold
  *
  */
```

Comment:
The empty javadoc will look a little funny.  Should we just delete it?  Or maybe attempt a very simple summary just to avoid the empty comment?

Reply:
There are many examples of this.. completely up to how we approach - there are probably dozens